mmv1/products/compute/api.yaml

```diff
@@ -388,6 +388,16 @@ objects:
                 scales up until it reaches the maximum number of instances you
                 specified or until the average utilization reaches the target
                 utilization.
+            - !ruby/object:Api::Type::String
+              name: 'predictiveMethod'
+              min_version: beta
+              default_value: NONE
```

Comment:
I think we'll need to apply `default_if_empty` on the field, as old autoscalers don't return `NONE` when it wasn't explicitly set. We could also remove the Terraform-side default, your call.

Reply:
Fixed